addons/html_builder/static/src/builder/options/border_configurator.xml

```diff
@@ -12,5 +12,9 @@
         </BuilderSelect>
         <BuilderColorPicker styleAction="getStyleActionParam('color')" t-if="state.hasBorder"/>
     </BuilderRow>
+
+    <BuilderRow t-if="props.withRoundCorner" label.translate="Round Corners" applyTo="props.applyTo">
+        <BuilderNumberInput styleAction="'border-radius'" unit="'px'"></BuilderNumberInput>
```

Comment:
`<BuilderNumberInput styleAction="'border-radius'" unit="'px'"/>` instead of
`<BuilderNumberInput styleAction="'border-radius'" unit="'px'"></BuilderNumberInput>` ?